tests/framework/extensions/provisioning/creates.go

```diff
@@ -552,6 +558,84 @@ func CreateProvisioningRKE1AirgapCustomCluster(client *rancher.Client, clustersC
 	return createdCluster, err
 }
 
+// CreateProvisioningAKSHostedCluster provisions an AKS cluster, then runs verify checks
+func CreateProvisioningAKSHostedCluster(client *rancher.Client) (*management.Cluster, error) {
+	cloudCredential, err := azure.CreateAzureCloudCredentials(client)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterName := namegen.AppendRandomString("akshostcluster")
+	clusterResp, err := aks.CreateAKSHostedCluster(client, clusterName, cloudCredential.ID, false, false, false, false, map[string]string{})
```

Comment:
I've got the same question for this one too

Reply:
Same as above.